qutebrowser/extensions/interceptors.py

```diff
@@ -29,6 +30,17 @@
     from PyQt5.QtCore import QUrl
 
 
+# Possible request types that can be received. Currently correspond to the
+# QWebEngineUrlRequestInfo Enum:
+# https://doc.qt.io/qt-5/qwebengineurlrequestinfo.html#ResourceType-enum
+ResourceType = enum.Enum('ResourceType',
```

Comment:
I think it'd be nicer to use the [class notation](https://docs.python.org/3/library/enum.html#creating-an-enum) for enums in the future (also, more tools are compatible with it).

This would also allow setting the names to the same values they have in QtWebEngine.